hie-plugin-api/hie-plugin-api.cabal

```diff
@@ -43,18 +49,21 @@ library
                      , fingertree
                      , free
                      , ghc
-                     , ghc-mod-core >= 5.9.0.0
+                     , hie-bios >= 0.3.2 && < 0.4.0
                      , ghc-project-types >= 5.9.0.0
-                     , haskell-lsp == 0.18.*
+                     , cabal-helper
+                     , haskell-lsp == 0.19.*
                      , hslogger
+                     , unliftio
```

Comment:
Unused dependency I think

Reply:
It seems to be used in: https://github.com/mpickering/haskell-ide-engine/blob/hie-bios/hie-plugin-api/Haskell/Ide/Engine/PluginsIdeMonads.hs#L644 or should this all be removed?

Reply:
We should either use `MonadBaseControl` OR `UnliftIO`, not both.